.github/workflows/release.yaml

```diff
@@ -21,6 +21,18 @@ jobs:
       file: images/svc-base/Dockerfile
       tag: ${{ github.event.inputs.tag }}-svc-base
 
+  build-dev-image:
```

Comment:
this is not actually a "dev" image, but the real image...
and it should be pushed as:
```
      namespace: ${{ github.event.repository.name }}
      name: ${{ github.event.repository.name }}
      tag: ${{ github.event.inputs.tag }}
```

Reply:
Just to be clear, "dev" is just the term I used.
This is the non-Federation image, which for 7/x branches is not used today in production (that's why I called it "dev").

I'm ok for pushing this image under `cloudserver/cloudserver:<tag>` when releasing (the Federation image is pushed under `cloudserver/cloudserver:<tag>-svc-base`)